css/style.css

```diff
@@ -508,19 +521,34 @@ summary:hover .summary-chevron-up svg {
 
 @media screen and (min-width: 768px) {
   .background {
-    background-image: url('../images/Header-bg-desktop.svg');
+    background-image: url("../images/Header-bg-desktop.svg");
```

Comment:
- [ ] Great job so far! 👏🏻  however according to the project requirement, on the desktop size the closed button(X), and the menu page must not be visible. Kindly correct this. You can use your @media screen and (min-width: 768px) to display none of your navigation menu page when you're on the desktop size. 😄

Reply:
![Capture](https://user-images.githubusercontent.com/92303891/173428938-6029ff51-62e9-4a6c-bb9f-bdbe345274ca.PNG)

Hello @BenMukebo  

Can you further explain the changes over there. As you can see the screenshot there is no visible closed button(X), and the menu page on the desktop version.

Reply:
Hello @Owusu-Desmond, 

What I mean, Let's try to open the mobile menu page when you're on the mobile size then try to increase your page until > 786px you are going to see that the navigation menu page will still be visible and showing up instead it close automatically